### PR TITLE
(#4216) - Use functionName() consistently

### PR DESCRIPTION
**Pull request by nickcolley**
Sorry, I should have tested your patch this is tested in an Windows 8 IE10 VM

### CODE REVIEW

**Comment by nickcolley** [comment]
See failing test: https://github.com/hoodiehq/pouchdb-hoodie-api/pull/75

**Comment by nolanlawson** [comment]
Whoops, +1 when green.

As a side note, I noticed that if/when we upgrade to leveldown 1.0.0+, the name will actually become `""`. This is fine, because MemDOWN will still be `"MemDOWN"`, but it's something to be aware of.

**Comment by nickcolley** [comment]
Hmm yeah this isn't very future proof. Maybe we should raise an issue to discuss something more concrete?

**Comment by nickcolley** [comment]
Also IE10 in Memory passed: https://travis-ci.org/pouchdb/pouchdb/jobs/78264903

Just restarting the anomalies.

**Comment by nickcolley** [comment]
All green :+1:

File: .travis.yml [hunk 1]
```diff
@@ -81,6 +81,7 @@ env:
   - CLIENT=saucelabs:chrome COMMAND=test
   - CLIENT=saucelabs:safari:6 COMMAND=test
   - CLIENT="saucelabs:internet explorer:10:Windows 8" COMMAND=test
+  - CLIENT="saucelabs:internet explorer:10:Windows 8" ADAPTERS=memory COMMAND=test
```
Comment (nolanlawson) [inline .travis.yml]: big +1 from me for this; I can close my other PR about adding a test

**Comment by nolanlawson** [comment]
Seems this green is a false positive:

> Not running test, cannot connect to saucelabs

Isn't it supposed to throw an error when this happens? Really misleading that all the tests show up as green. The only reason I noticed is that the builds were taking like ~1 minute, which is way too fast.

**Comment by daleharvey** [comment]
Hrm, it passing if the saucelabs credentials arent given is on purpose, otherwise all PR's from non org members would all fail saucelabs, however @nickcolley is a pouchdb org member so should get saucelabs permission. I think you may need to create the PR from a branch on the pouchdb org instead of your own fork to get them

**Comment by daleharvey** [comment]
so `git push pouchdbrepo 4216-use-functionName-consistently' and create the PR from that

**Comment by nickcolley** [comment]
Does this mean all PRs are false positives from saucelabs?

**Comment by daleharvey** [comment]
All PR's that arent made from branches off the PouchDB repo yup, there is an open issue @ https://github.com/pouchdb/pouchdb/issues/3203 for it

**Comment by daleharvey** [comment]
doing a full run @ https://github.com/pouchdb/pouchdb/pull/4264

**Pull request closed without merging**